packages/common/src/directives/ng_switch.ts

```diff
@@ -149,10 +145,9 @@ export class NgSwitch {
   }
 
   private _updateDefaultCases(useDefault: boolean) {
-    if (this._defaultViews && useDefault !== this._defaultUsed) {
+    if (useDefault !== this._defaultUsed) {
```

Comment:
```suggestion
    if (this._defaultViews.length > 0 && useDefault !== this._defaultUsed) {
```
I think you need to have the `length` check here to be equivalent with how it was before.

Reply:
Done. I also cleaned up the commits to match 1 file changed/per commit.